scripts/ci/ensure-angular-dependencies-compatibility.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
> Not sure what's the equivalent to this -A option.

Oh damn. `declare -A` creates an associative array and is a new feature in Bash 4, so I guess the bash version in OS X must be older than that. I'll have to rewrite it using two linear arrays instead of a single associative array. No biggie.

> I pushed it here so we can see if the CI servers complain with another error.

https://travis-ci.org/resin-io/etcher/jobs/209813569 fails with the same error, **but** TravisCI thinks the job passed! Eeek, do we need to tweak something somewhere so that if `npm run sanity-checks` fails then the whole CI job fails?

> Feel free to push straight to this PR btw.

Cool will do. I think we ought to fix the non-failing-job problem first though, before I update the PR ;-)